src/client.py

```diff
@@ -0,0 +1,68 @@
+import requests
+import matplotlib.pyplot as plt
+import sys
+from errors import WrongSettlementException, InvalidSettlement
+
+
+
+def display_alerts_distribution(settlement:str, host:str, port:str):
+    param = {"settlement":settlement}
+    url = f"http://{host}:{port}/get_distribution"
+    
+    response = requests.get(url, params= param)
+    
+    if response.status_code == 200:
+        # in case the settlement exist but no alarms were there
+        if "message" in response.json():
+            sentence = response.json().get('message', '')
+            print(sentence)
+        else:    
+            response_list = response.json()
+            hour_vector = [d['hour'] for d in response_list]
+            alerts_count_vector = [d['count'] for d in response_list]
+            # Create a bar chart
+            plt.bar(hour_vector, alerts_count_vector, color='#E84D88')
+            # Rotate x-axis labels vertically
+            plt.xticks(rotation='vertical')
+            plt.xlabel('Hour')
+            plt.ylabel('Alerts Amount')
+            plt.title(f'Alerts Hourly Distribution in {settlement}')
+            plt.show()
+    elif response.status_code == 400:
```

Comment:
You dont need this duplication, especially if its all the same line of code!.
you can use a built in function on `response.raise_for_status()` for that.